hls4ml/utils/config.py

```diff
@@ -79,8 +79,13 @@ def _get_precision_from_quantizer(quantizer):
         rnd = "AP_RND_CONV"
         overflow = "AP_SAT"
         if quantizer['class_name'] in ('quantized_relu', 'quantized_relu_po2'):
-            signed = False
-            integer -= 1
+            if quantizer['config']['negative_slope'] != 0.0:
+                print(quantizer)
```

Comment:
get rid of the print statement and commented out code

Reply:
Print and commented code removed